src/components/pages/enterprise/training/training.jsx

```diff
@@ -3,12 +3,21 @@ import React from 'react';
 import Button from 'components/shared/button';
 import Container from 'components/shared/container';
 import Heading from 'components/shared/heading';
+import AcendLogo from 'icons/logo-acend.inline.svg';
 import IsovalentLogo from 'icons/logo-isovalent.inline.svg';
 
 const title = 'Training';
 const description =
   'The following partners offer Cilium related training in accordance with the training partner requirements of the Cilium project.';
 
+const acend = {
+  title: 'Cilium Basics',
+  description:
+    'This one-day training in English or German provides a practical and clear introduction to Cilium, an eBFP-based networking, observability and security stack for cloud and on-premise infrastructure.',
+  buttonText: 'Learn more',
+  buttonLink: 'https://www.acend.ch/en/trainings/cilium/',
```

Comment:
Let's add target="_blank" to this button since `acend` is the external URL.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/A#attr-target